gen/combine_gdal_doxygen_xml.py

```diff
@@ -0,0 +1,37 @@
+# combines alls xmls
+# place in directory with all xmls created by doxygen
+# this is by default gdal/xml/
+
+# done previously by running combine.xslt
+# however the index.xml doesn't seem up to date
+# with the contents of gdal/xml/
+
+# after this script, doxygen.xml will be around 80 MB
+# I manually deleted everything not under
+# sectiondef, briefdescription, and detaileddescription
```

Comment:
I used XMLSpy for this, but this is not free software. I'll add some code to the end of the python script that does the same. Note that this step is not essential, but it does speed up the wrapping since it takes less time to lookup documentation.